config/routes.rb

```diff
@@ -3,4 +3,11 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/guests/:id", to: "guests#show"
+  post "/guests/:id", to: "rooms_guests#create"
```

Comment:
This route is not restful, create routes do not end with ID's. The verb, controller, and action are correct, the path should be: `"/guests/:id/rooms"`

Remember that we are creating a link between a guest and a room, not creating a guest.